ckanext/data_qld/resource_freshness/validation.py

```diff
@@ -52,7 +52,7 @@ def validate_next_update_due(keys, flattened_data, errors, context):
             today = dt.datetime.now(h.get_display_timezone())
             if next_update_due.date() <= today.date():
                 errors[keys].append(_("Valid date in the future is required"))
-        elif data_qld_helpers.is_api_request():
+        elif data_qld_helpers.is_api_request() or not current_next_update_due:
```

Comment:
it it not also breaking on https://github.com/qld-gov-au/ckanext-data-qld/blob/c6a103dcbcff6bea67338324c989c41f35973e86/ckanext/data_qld/resource_freshness/validation.py#L90

Reply:
Not anymore because `package_show` and resource_show` will now return the `nature_of_change` value which will make it pass that validation check.
The above change is for this story https://salsadigital.atlassian.net/browse/DQL3-75

Reply:
cool, lets get this to our repo and see if it makes staging happy again.